### DISCUSSION TITLE
V9 and Picker

### DISCUSSION THREAD

**Discussion by PooLP** [microsoft/fluentui#23429] [Q&A] [answered]
Hi,
The v9 do not have any Picker ? (People Picker, Color Picker ...)

**Comment by khmakoto** [comment, marked as answer]
Hi @PooLP, not yet, v9 includes a first set of components since it is basically a revamp of the library. We will include more as time goes on. However, v9 is created in such a way that you can use v8 and v9 side by side in case there is something in v8 that does not exist in v9 yet.
